### ISSUE TITLE
Restrictive keyboard access

### ISSUE THREAD

**Issue by JoelHaslam** [grafana/grafana#39805]
When using tab to navigate the UI, you are unable to get keyboard focus into any of the panel available on the dashboard. This limits the user quite heavily, meaning that they are unable to edit the panel, plus more.

**Steps to reproduce**:
On the Grafana UI, press tab to cycle through all the options and attempt to get keyboard focus of any of the panels available.

**Actual Result**:
Unable to get focus of an individual panel. However if the panel contains a table that has links in them (or any sort of element that attracts keyboard focus), you are able to access that.

e.g:
![image](https://user-images.githubusercontent.com/63413067/135305915-7f99120b-2d85-4ecf-8cd3-36514e7e6c98.png)

**Expected Result**:
Once the keyboard focus has cycled around each of the icons, it would begin to cycle through each of the panels and allow you to access the keyboard shortcuts available after manually selecting a graph.

**Relevant WCAG Criteria:** [2.1.1 keyboard](https://www.w3.org/TR/WCAG21/#keyboard)

**Environment**:
- Grafana version: 8.1.4
- Data source type & version: Default data used at [play.grafana.org](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1)
- User OS & Browser: Windows & Chrome
- Others:

**Comment by torkelo** [comment]
We have project that is improving keyboard navigation and other acccessability issues and making good progress, check `accessability` label for current issues and epics to track progress. 

This one is covered in this issue 
https://github.com/grafana/grafana/issues/36776
